dateparser_cli/fasttext_manager.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+import urllib.request
+
+from .utils import date_parser_model_home
+
+
+def fasttext_downloader(model=[]):
+    model_url = {
+        "small": "https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.ftz",
+        "large": "https://dl.fbaipublicfiles.com/fasttext/supervised-models/lid.176.bin"
+    }
+
+    model_name = "small"
+    if model:
+        if model[0] == "large":
+            model_name = "large"
```

Comment:
I don't think it is the correct behavior, because if you put a wrong model name it takes the `small`. Example: ` dateparser-download fasttext aaaaaa`. I think this would be better:

```suggestion
    if not model:
        model_name = "small"
    elif model and model[0] == "large":
        model_name = "large"
    else:
        print("Couldn't find a model called \"{}\". Supported models are:"
              " {}".format(model[0], ", ".join(model_url.keys())))
        return
```

Reply:
BTW, change the `print`s to `log`s.

Reply:
I changed `prints` to `logs`.